packages/flame/lib/src/components/component_track_component.dart

```diff
@@ -0,0 +1,38 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/text.dart';
+
+class ComponentTrackComponent<T> extends TextComponent {
```

Comment:
This is a bit of a confusing name :sweat_smile:

Reply:
Yes, this is so freaking confusing hahaha, but like I just submitted this as it was based on my perfomance debugging, and I was not really caring about names hahahahaha

Maybe `ChildTrackComponent`? Since this will track the number of a certain child of a parent?

Reply:
Ngl, the first time I read this I thought it was about a race car game! Since I interpreted "track" as the "race track" 😂

Reply:
> Maybe ChildTrackComponent?

I don't think that really conveys what it does either, maybe `ChildCounterComponent` or something like that?